service/common/pip_installer.py

```diff
@@ -31,8 +32,12 @@
 
 
 class ModuleInfo(NamedTuple):
+  """PIP module installation information and constraints."""
   # Name of the module to install.
   pip_module_name: str
+  # Name of the optional module to import to determine whether the required
+  # pip package is installed.
+  import_module_name: str = ''
```

Comment:
maybe explain when this is needed? it is otherwise a bit confusing that whenever this is set (in the values below), it matches pip_module_name

Reply:
Done, PTAL